### ISSUE TITLE
Selection has issue with translated labels

### ISSUE THREAD

**Issue by fnagel** [fabarea/vidi#90]
Using built-in fe_user module:  When saving a selection with English backend and try reloading this selection with another BE user with different BE language (German for example) the property is ignored while filtering the list. 

Try filtering for a specific email to reproduce this issue (at least with German and English backend language).

It seems its using the speaking_query for populating the selections.

tl;dr; Translated property labels in selections seems to break list filtering.

Tested in TYPO3 6.2.15 and vidi 1.3.1

**Comment by fnagel** [comment]
Workaround: Override TCA grid -> columns -> FIELD -> label with a single label string for all languages.

**Comment by fabarea** [comment]
Mmm... I had more thoughts about this pending issue. It turns out, It is not trivial to support localization in speaking "query" (Visual Search bar). It would require some deep refactoring and I am leaving this issue aside for now as a known bug.
